lib/src/merge.rs

```diff
@@ -816,6 +882,100 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_deduplicate() {
+        // 1-way merge
+        assert_eq!(c(&[], &[0]).deduplicate(), c(&[], &[0]));
+        // 3-way merge
```

Comment:
Optional: I'd prefer "2-sided merge" here.

Reply:
That would make it inconsistent with the other tests in this file. I don't mind either term, but I prefer it to be consistent within the file.